src/pages/components/product-form.tsx

```diff
@@ -311,76 +219,11 @@ const ProductForm = ({
                       type="text"
                       id="t"
                       name="t"
-                      value={formValues.t}
+                      value={getFormValue('t')}
                       placeholder="Optional"
                       onChange={handleChange}
                       className="w-full p-2 border border-gray-300 rounded"
                     />
-
```

Comment:
Getting rid of this. If in the future we want to reimplement this stuff. we can reference previous commits